ext/NonlinearSolveSIAMFANLEquationsExt.jl

```diff
@@ -0,0 +1,171 @@
+module NonlinearSolveSIAMFANLEquationsExt
+
+using NonlinearSolve, SciMLBase
+using SIAMFANLEquations
+import ConcreteStructs: @concrete
+import UnPack: @unpack
+import FiniteDiff, ForwardDiff
+
+function SciMLBase.__solve(prob::NonlinearProblem, alg::SIAMFANLEquationsJL, args...; abstol = 1e-8,
+        reltol = 1e-8, alias_u0::Bool = false, maxiters = 1000, termination_condition = nothing, kwargs...)
```

Comment:
Is this the same default we are using elsewhere?

Reply:
The `abstol` and `reltol` are determined based on the problem eltype

Reply:
Yeah, ```abstol``` and ```reltol``` should just be determined from the problem eltype